src/dynamoService.ts

```diff
@@ -0,0 +1,124 @@
+import { DynamoDB } from "@aws-sdk/client-dynamodb";
+import {
+  DynamoDBDocumentClient,
+  PutCommand,
+  GetCommand,
+  UpdateCommand,
+  DeleteCommand,
+  QueryCommand,
+} from "@aws-sdk/lib-dynamodb";
+
+const dynamoDBClient = new DynamoDB({ region: "your-region" });
+const dynamoDBDocClient = DynamoDBDocumentClient.from(dynamoDBClient);
+
+class DynamoService {
+  private tableName: string;
+
+  constructor(tableName: string) {
+    this.tableName = tableName;
+  }
+
+  // Create
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
The procedure that I would like to enforce for eslint is no inline disabling of rules. If you have a case for explicit any, then make it. And then we can allow it globally.

Reply:
Assuming you actually have a good reason

Reply:
I originally tried to create this to be as much of a modular service class as possible, so that it could be reused in as broad a range of scenarios as possible. Your suggestion lead to a much more modular service class overall, where all we'd have to use it elsewhere is change the type and possibly import and implement UpdateCommand. Long story short—I removed the suppression and added the specific type.